### DISCUSSION TITLE
Allow disabling nightcore beats

### DISCUSSION THREAD

**Discussion by huupoke** [ppy/osu#14209] [Ideas]
Sometimes, the nightcore beats doesn't really suit the songs style, so I suggest that there is an option to disable it, while still keeping the increased pitch effect.

**Comment by goodtrailer** [comment]
You can remove nightcore beats through skinning, though I guess it could be somewhat inconvenient if you want to change it between maps.

**Comment by zachmanthethird** [comment]
If nothing else, DayCore does not provide a backtrack while NightCore does.
The two should operate the same regardless of the decision made here.

**Comment by verarr** [comment]
This is probably solved now, with _Double Time_'s _Adjust pitch_ option. Unless someone wants to have non-1.5x speed with 1.5x pitch adjustment _and_ no nightcore beats.

Alternative suggestion: merge NC into DT and make nightcore beats a customization option.

**Reply by rushiiMachine** [reply]
The issue with DT's adjust pitch is that it doesn't preserve constant pitch and is audibly different from the pitch adjustment nightcore does.
